tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
-    "target":"ES2018",
+    "target": "ES2018",
     "module": "commonjs",
-    "lib": ["es2016", "es2017.object", "es2017.string"],
+    "lib": ["es2016", "es2017.object", "es2017.string", "dom"],
```

Comment:
How come we need `dom` as well?

Reply:
I was seeing 
```
npm ERR! provider/awsCdkProvider.ts(52,7): error TS2584: Cannot find name 'console'. Do you need to change your target library? Try changing the `lib` compiler option to include 'dom'.
```
but maybe just some fluke